lib/active_merchant/billing/gateways/kushki.rb

```diff
@@ -204,6 +206,33 @@ def add_deferred(post, options)
         }
       end
 
+      def add_three_d_secure(post, payment_method, options)
+        three_d_secure = options[:three_d_secure]
+        return unless three_d_secure
+
+        post[:threeDomainSecure] = {}
+        post[:threeDomainSecure][:eci] = three_d_secure[:eci]
+        post[:threeDomainSecure][:specificationVersion] = three_d_secure[:version]
```

Comment:
💬 Comment:

Just matter of taste but initialization could be more declarative instead of procedural:

```ruby

post[:threeDomainSecure] = {
  eci: three_d_secure[:eci],
  specificationVersion: three_d_secure[:version]
}
```

Reply:
Noted! Thanks!